src/_scss/components/_accordionCheckbox.scss

```diff
@@ -0,0 +1,64 @@
+.filter-item-wrap {
+  padding: rem(20) rem(32);
+
+  .checkbox-filter__wrapper {
+    .checkbox-filter__header {
+      font-size: $font-size-12;
+      line-height: 1.5;
+      display: flex;
+      align-items: center;
+
+      svg {
+        color: $blue-50;
+        max-width: rem(10);
+        max-height: rem(10);
+      }
+
+
+      input {
+        margin-left: rem(8);
```

Comment:
should use the var for global-margin for these

Reply:
global margin variable used to replace generic rems